packages/addons/service/tvheadend/package.mk

```diff
@@ -0,0 +1,48 @@
+################################################################################
+#      This file is part of LibreELEC - https://libreelec.tv
+#      Copyright (C) 2017-present Team LibreELEC
+#
+#  LibreELEC is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 2 of the License, or
+#  (at your option) any later version.
+#
+#  LibreELEC is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with LibreELEC.  If not, see <http://www.gnu.org/licenses/>.
+################################################################################
+
+PKG_NAME="tvheadend"
+PKG_VERSION="1.0"
+PKG_REV="100"
+PKG_ARCH="any"
+PKG_LICENSE="GPL"
+PKG_SITE=""
+PKG_URL=""
+PKG_DEPENDS_TARGET="toolchain"
+PKG_SECTION="service"
+PKG_SHORTDESC="Add-on removed"
+PKG_LONGDESC="Add-on removed"
+PKG_AUTORECONF="no"
+
+PKG_ADDON_BROKEN="Tvheadend 4.0 is no longer maintained and has been superseded by Tvheadend 4.2."
```

Comment:
`PKG_ADDON_BROKEN=""` should be added [here](https://github.com/LibreELEC/LibreELEC.tv/blob/libreelec-8.2/config/path#L76), otherwise any package.mk loaded after this one will also be broken.

`PKG_ADDON_TYPE=""` and `PKG_ADDON_NAME=""` should also probably be added for the same reason.

Reply:
added `config/path: ensure package.mk variables are unset` commit

Reply:
Looks good, thanks.